examples/profile/main.go

```diff
@@ -64,13 +65,45 @@ func profile(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	rootdir, _ := os.Getwd()
+	// Check if the cert files are available.
+	err := certificatePresenceCheck("yotiSelfSignedCert.pem", "yotiSelfSignedKey.pem")
+	// If they are not available, generate new ones.
+	if err != nil {
+		err = generateSelfSignedCertificate("yotiSelfSignedCert.pem", "yotiSelfSignedKey.pem", "127.0.0.1:8080")
+		if err != nil {
+			log.Fatal("Error: Couldn't create https certs.")
```

Comment:
I don't know Go, but.... will this try to continue executing, after it's done a log.Fatal?

Reply:
No it won't continue - it's in effect a call to `Print()` followed by a call to `os.Exit(1)`.